test/integration/apiserver/apply/apply_test.go

```diff
@@ -1212,7 +1212,7 @@ func TestClearManagedFieldsWithMergePatch(t *testing.T) {
 		Namespace("default").
 		Resource("configmaps").
 		Name("test-cm").
-		Body([]byte(`{"metadata":{"managedFields": [{}]}}`)).Do(context.TODO()).Get()
```

Comment:
do we have tests exercising clearing managed fields using all of the following scenarios:
* resource types: built-in and custom
* patch types: merge, json, and strategic-merge
* managedFields content: `[{}]` and `[]`